src/core/fbs/fbs-adapter.yaml

```diff
@@ -0,0 +1,1712 @@
+swagger: '2.0'
+info:
+    title: 'FBS Adapter'
+    version: '1.0'
+basePath: /
+paths:
```

Comment:
Please add some from of documentation regarding how we ended up with this local file and how it should be maintained.

I image this could either be in the form of a markdown file in the `docs` dir or a code comment at the top of the file.

Reply:
Added document with information